helm-chart/renku-graph/charts/jena/templates/lucene-config.yaml

```diff
@@ -11,43 +11,44 @@ metadata:
 data:
   {{ .Values.global.graph.jena.dataset }}.ttl: |-
 
-    @prefix :        <#> .
-    @prefix fuseki:  <http://jena.apache.org/fuseki#> .
-    @prefix rdf:     <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
-    @prefix rdfs:    <http://www.w3.org/2000/01/rdf-schema#> .
-    @prefix tdb:     <http://jena.hpl.hp.com/2008/tdb#> .
-    @prefix ja:      <http://jena.hpl.hp.com/2005/11/Assembler#> .
-    @prefix text:    <http://jena.apache.org/text#> .
+                                                 @prefix:        <#> .
```

Comment:
is this indented so far on purpose?

Reply:
No this is a side effect of the scala formatter.